docs/modules/commons/pages/expressions.adoc

```diff
@@ -294,6 +294,50 @@ Compress the input string to GZip and encode compressed bytes to Base64 format
 |`H4sIAAAAAAAAACvLLMtMKS0GANIHCdkHAAAA`
 |===
 
+== Hash calculations
+
+=== `calculateHash`
+
+Calculates the hash using the specified hashing algorithm
+
+[source, subs="+quotes"]
+----
+#{calculateHash($algorithm, $input)}
+----
+
+:algorithm-summary: hash algorithm (can use https://datatracker.ietf.org/doc/rfc1319[MD2], https://datatracker.ietf.org/doc/rfc1321[MD5], https://datatracker.ietf.org/doc/rfc3174[SHA-1], https://datatracker.ietf.org/doc/rfc6234[SHA-256], https://datatracker.ietf.org/doc/rfc6234[SHA-384], https://datatracker.ietf.org/doc/rfc6234[SHA-512])
```

Comment:
it's better to move it under `== Hash calculations` to show its usage scope better